BitwardenShared/UI/Auth/StartRegistration/StartRegistrationState.swift

```diff
@@ -0,0 +1,49 @@
+import SwiftUI
+
+// MARK: - StartRegistrationState
+
+/// An object that defines the current state of a `StartRegistrationView`.
+///
+struct StartRegistrationState: Equatable {
+    // MARK: Properties
+
+    /// The text in the email text field.
+    var emailText: String = ""
+
+    /// Whether the terms and privacy toggle is on.
+    var isReceiveMarketingToggleOn: Bool = false
+
+    /// The text in the name text field.
+    var nameText: String = ""
+
+    /// The text in the receive marketing emails toggle
+    var receiveMarketingEmailsText: String {
+        Localizations.getEmailsFromBitwardenForAnnouncementsAdviceAndResearchOpportunitiesUnsubscribeAtAnyTime
+            .replacingOccurrences(
+                of: Localizations.unsubscribe,
+                with: "**[\(Localizations.unsubscribe)](\(ExternalLinksConstants.unsubscribe))**"
+            )
+    }
```

Comment:
⚠️ As with `goBackText` you need to be careful here in case some translation is done different when `Unsubscribe` is used alone vs in a sentence. French may be a good example here.

Reply:
For now clickable texts are going to be used with markup in translation files and a variable to inject the URL.